### PR TITLE
Add Texas, USA provider

### PR DESCRIPTION
**Pull request by oja**
Provider for Houston, Dallas, San Antonio, Fort Worth, and Austin.

I tried to run gradle tests, but came up with some errors I suspect were caused due to not running with proper android support. Please let me know if I messed anything up.

### CODE REVIEW

**Comment by OjasA** [comment]
OK, based on TravisCI it looks like the problem was just on my system.

**Comment by grote** [comment]
Hey @OjasA, thanks a lot for your PR! :)
I think Travis does not run the live tests, so it is possible the the problem was not only on your system.

**Comment by OjasA** [comment]
I keep getting an error about how the constructors in TexasProviderLiveTest don't have the proper return type (although they don't need one). Is this the issue that you're getting too?

File: enabler/src/de/schildbach/pte/TexasProvider.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+public class TexasProvider extends AbstractNavitiaProvider
+{
+	private static String API_REGION = "us-tx";
+
+	public TexasProvider(final String apiBase, final String authorization)
+	{
+		super(NetworkId.TX, apiBase, authorization);
+
+		setTimeZone("America/Chicago");
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/TexasProvider.java]: Timezone Chicago, really? Isn't there a major city in Texas?

File: enabler/src/de/schildbach/pte/NetworkId.java [hunk 2]
```diff
@@ -77,7 +77,7 @@
 	JET,
 
 	// United States
-	SF, SEPTA,
+	SF, SEPTA, TX,
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/NetworkId.java]: I think the ID should stay in sync with the name of the provider. So TEXAS.

**Comment by schildbach** [comment]
Thanks for your contribution. I have added some comments. Would you like to rebase to current master? And maybe adapt the year of copyright to 2016?

File: enabler/src/de/schildbach/pte/TexasProvider.java (hunk 1 above)
Comment (OjasA) [inline enabler/src/de/schildbach/pte/TexasProvider.java]: @schildbach this was intentional. The string passed to setTimeZone appears to be based off of [the tz format](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), which only includes Chicago for Central Time in the US (UTC-5).

**Comment by OjasA** [comment]
@schildbach OK, I addressed your suggestions regarding licensing and switching the name of the ID to match with the provider. I didn't change the time zone for the reason stated in the comment. 

Otherwise, commit is rebased and should be good to go!

**Comment by oja** [comment]
Hello @schildbach,

Would it be possible to get this merged sometime soon?

**Comment by xamanu** [comment]
I tested this today:

* There is a conflict in this branch. Some minor issue's in the `NetworkID.java`.
* I had problems running a build with gradle: `/home/jitpack/build/enabler/src/de/schildbach/pte/TexasProvider.java:33: error: no suitable constructor found for AbstractNavitiaProvider(NetworkId,String,String)`

**Comment by oja** [comment]
It seems that I deleted my fork sometime within the last two years, so I'm going to close this PR and open a new, rebased one.

**Pull request closed without merging**